lib/rubocop/cop/lint/format_parameter_mismatch.rb

```diff
@@ -136,7 +136,11 @@ def expected_fields_count(node)
             .source
             .scan(FIELD_REGEX)
             .reject { |x| x.first == PERCENT_PERCENT }
-            .reduce(0) { |acc, elem| acc + (elem[2] =~ /\*/ ? 2 : 1) }
+            .reduce(0) { |acc, elem| acc + arguments_for_wildcard(elem[2]) }
+        end
+
+        def arguments_for_wildcard(format)
```

Comment:
The name of this method is extremely misleading IMO - after all we're using it to process everything, not just wildcards.